### ISSUE TITLE
Quick fix of listenerMiddleware.startListening.withTypes method

### ISSUE THREAD

**Issue by StanislavDalinin** [reduxjs/redux-toolkit#4378]
Hi there,
Can we change this [line](https://github.com/reduxjs/redux-toolkit/blob/e46eb99f135dde5f047b8dc09113c42f20a54353/packages/toolkit/src/listenerMiddleware/types.ts#L695)  in [redux-toolkit/packages/toolkit/src/listenerMiddleware/types.ts](https://github.com/reduxjs/redux-toolkit/packages/toolkit/src/listenerMiddleware/types.ts) to this
```
  withTypes: <
    OverrideStateType extends StateType,
    OverrideDispatchType extends ReduxDispatch = ThunkDispatch<
      OverrideStateType,
      unknown,
      UnknownAction
    >,
  >() => TypedStartListening<OverrideStateType, OverrideDispatchType, ExtraArgument>
``` 
just add `ExtraArgument`, to have it inherited.
Because for now, the `ExtraArgument` type is losing while creating like
```
export const startAppListening = listenerMiddleware.startListening.withTypes<
  RootState,
  AppDispatch
>()
```

**Comment by markerikson** [comment]
I'm not sure what you're trying to describe here. Can you clarify this?

**Comment by StanislavDalinin** [comment]
Hi, thank you for your attention!
That issue was fixed in the 2.2.7 release.

> listenerMiddleware.withTypes() methods now allow passing in an ExtraArgument generic.

We can close that thread. Thank you for your effort!
